hwilib/devices/bitbox02_lib/README.md

```diff
@@ -1,12 +1,12 @@
 # Python BitBox02 Library
 
-This is a slightly modified version of the official [bitbox02](https://github.com/digitalbitbox/bitbox02-firmware/tree/master/py/bitbox02) library.
+This is a slightly modified version of the official [bitbox02](https://github.com/BitBoxSwiss/bitbox02-firmware/tree/master/py/bitbox02) library.
```

Comment:
Looks like the Github repo moved without a redirect. I checked on https://bitbox.swiss/dev/ that the new org account is correct.

Reply:
What do you mean by 'without a redirect'?

ttps://github.com/digitalbitbox/bitbox02-firmware/tree/master/py/bitbox02 redirects to ttps://github.com/BitBoxSwiss/bitbox02-firmware/tree/master/py/bitbox02